velox/functions/prestosql/aggregates/AggregateNames.h

```diff
@@ -23,6 +23,7 @@ const char* const kApproxMostFrequent = "approx_most_frequent";
 const char* const kApproxPercentile = "approx_percentile";
 const char* const kApproxSet = "approx_set";
 const char* const kArbitrary = "arbitrary";
+const char* const kAnyValue = "any_value";
```

Comment:
This is a list of Presto functions. Presto doesn't have any_value, hence, should not be added here.

MySQL functions are different from Presto. You may need to create a separate function package for these. Re-using Presto functions will cause a bunch of issues.

Reply:
Thanks, I got it.

Reply:
Since the aggregate function any_value have been added to Presto in https://github.com/prestodb/presto/pull/21389 ,  can we continue this PR?